bundle/manifests/self-node-remediation.clusterserviceversion.yaml

```diff
@@ -41,7 +41,19 @@ metadata:
     description: Self Node Remediation Operator for remediate itself in case of a
       failure.
     olm.skipRange: '>=0.4.0'
-    operatorframework.io/suggested-namespace: openshift-workload-availability
+    operatorframework.io/suggested-namespace-template: |-
```

Comment:
we need to keep `operatorframework.io/suggested-namespace`, the template is new OCP 4.13